### PR TITLE
Add/Remove (& Local Storage) feature

### PR DESCRIPTION
**Pull request by Botlhale-Setou**
- Add code for add button.
- Add code for remove button.
- Ensure proper array item indexing.
- Incorporate local storage.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -61,10 +63,40 @@ const refreshList = () => {
...
+
+  const localStorageItem = window.localStorage.getItem('tasks');
+
+  if (localStorageItem) {
+    tasks.arrToDos = JSON.parse(localStorageItem);
+  }
+
+  refreshList();
+});
+
```
Comment (Lucas-Masaba) [inline src/index.js:81]: - [ ] Please add a button to remove a single task as requested. As ofnow, I can only delete the completed tasks

File: src/ToDoLibrary.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import ToDo from './ToDo.js';
```
Comment (Lucas-Masaba) [inline src/ToDoLibrary.js:1]: - [ ] Please add a function that enables a user to edit the text of the tasks as requested in the instructions. 👍

**Review by Lucas-Masaba — CHANGES_REQUESTED** [review]
# Hello @Botlhale-Setou  👋 , 

Good job so far! 👍 
There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there

## Highlights 🍾 
- Linter checks passing ✔️ 
- Most of the functionality is working 👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 💭 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

- [x] When I click a checkbox to mark a task as complete, the complete value is not changed from false to true and when I refresh he page, the checked box is not saved. Please fix that.

File: src/ToDoLibrary.js (hunk 2 above)
Comment (Botlhale-Setou) [reply src/ToDoLibrary.js:1]: Done! 👍

**Review by frank1738 — APPROVED** [review]
Hi @Botlhale-Setou , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

YesYouAreGIF (2)

## Highlights

✅ Correct use of Github workflow
✅ No linter error
✅ README is okay
✅ App is works as expected 👍 

## Optional suggestions

N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**